frontend/src/leftBar.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * \brief : React module that encapsulates the left side of the UI 
+ *          containing the canvas and some button functionnalities
+ */
+
+import React from "react";
+import { Component } from "react";
+import { View, Text } from "react-native";
+import { leftBarStyle } from "../styles/leftBarStyle";
```

Comment:
Est-ce qu'il y a moyen de dealer avec les relative imports? 
J'ai trouvé ça pour la config d'un fichier src global:
https://plainenglish.io/blog/why-and-how-to-use-absolute-imports-in-react

Ça donnerait genre ça pour la ligne:
`import { leftBarStyle } from "styles/leftBarStyle";`

Reply:
Je pense que ça implique que tout va être dans `src` (styles, templates, etc.)?